system/BaseModel.php

```diff
@@ -1291,10 +1291,10 @@ protected function intToDate(int $value)
                 return $value;
 
             case 'datetime':
-                return date('Y-m-d H:i:s', $value);
+                return (string) Time::createFromTimestamp($value, app_timezone());
```

Comment:
The `$value` is the current timestamp from `setDate()`.
It comes from `Time::now()`.
So I don't see a need to call `Time` again.